src/components/strings/index.js

```diff
@@ -1,104 +1,103 @@
-export const uploadDesktop = {
+export const uploadDesktop = (i18n) => ({
```

Comment:
why can't use just the local file straight away?
A lot of redundancy redeclaring everything again.

Reply:
That's what I did in my first implementation but the keys were becoming heavily interpolated and I wasn't sure it was a good idea to delegate conditional logic to the presence of a key/copy, which made decided for this solution. At a closer look, I think having this redundancy may generate bugs in the long run, therefore I will find a better way to handle the interaction between copy and logic.